website/ref/language.md

```diff
@@ -1575,6 +1575,14 @@ Redirections may appear anywhere in the command, except at the beginning,
 although this may be restricted in future. It's usually good style to write
 redirections at the end of command forms.
 
+**Important:** Elvish only supports reading and writing bytes from/to the target
```

Comment:
I'd use "Note" rather than "Important" for consistency with the rest of the doc.